lighthouse-core/lib/traces/pwmetrics-events.js

```diff
@@ -182,8 +182,8 @@ class Metrics {
     }
 
     // confirm our navStart's correctly match
-    const navStartTs = metrics.find(e => e.id === 'navstart').ts;
-    if (this.getNavigationStartEvt().ts !== navStartTs) {
+    const navStartEvt = metrics.find(e => e.id === 'navstart');
+    if (!navStartEvt || this.getNavigationStartEvt().ts !== navStartEvt.ts) {
```

Comment:
yup it should always find something.

The `getNavigationStartEvt` method could live on top of trace-of-tab as well, though...